admin/doc-requirements.txt

```diff
@@ -1,4 +1,4 @@
-Sphinx == 1.6.3
+Sphinx == 1.8.3
 git+https://github.com/ceph/sphinx-ditaa.git@py3#egg=sphinx-ditaa
 git+https://github.com/michaeljones/breathe#egg=breathe
-pyyaml==3.13
+pyyaml>=4.2b1
```

Comment:
At the time of this writing, 4.2 is unreleased (pyyaml homepage doesn't describe the contents of the 4.2* series). Although there is a 4.2b4 now, I think it would be useful to add a note that the usage of a beta release is needed here to address the CVE

Reply:
@alfredodeza shall i put the note in doc-requirements.txt in addition to the commit message?

Reply:
The commit message is fine as-is I think, I still had to look for 4.2 to end up understanding why you were using a beta release (4.2 is unreleased but has the fix). So a comment in the requirements.txt would work great

Reply:
done.